src/main/java/core/basesyntax/Plane.java

```diff
@@ -1,5 +1,70 @@
 package core.basesyntax;
 
 public class Plane {
+    private String model;
+    private int capacity;
+    private String type;
+    private int year;
+    private int range;
+
+    private Plane(PlaneBuilder builder) {
+        this.model = builder.model;
+        this.capacity = builder.capacity;
+        this.type = builder.type;
+        this.year = builder.year;
+        this.range = builder.range;
+    }
+
+    @Override
+    public String toString() {
+        return "Plane{"
+                + "model='" + model + '\''
+                + ", capacity='" + capacity
+                + '\''
+                + ", type='"
+                + type + '\''
```

Comment:
```suggestion
                + ", capacity='" + capacity + '\''
                + ", type='" + type + '\''
```
same for other fields

Reply:
fixed